src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleFormUtility.ts

```diff
@@ -36,9 +36,9 @@ export function getConditionSecondaryText(
     values: unknown[]
 ): string {
     if (type === 'NumericRange') {
-        return `${values[0]} ${i18n.t('min')} ${i18n.t('to')} ${
+        return `${values[0]} (${i18n.t('min').toLowerCase()}) ${i18n.t('to')} ${
```

Comment:
can't we just use t from useTranslation? is the i18n.t will get the current selected language from context? It happened to me once that is why I just wanted to make sure you tried it :)